docs/csharp/tutorials/intro-to-csharp/arrays-and-collections.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Collections tutorial - C# local quickstarts
+title: Collections tutorial - Introduction to C#
 description: Learn C# by exploring the List collection in this tutorial.
 ms.date: 10/13/2017
 ms.custom: mvc
 ---
-# C# Quickstart: Collections
+# Introduction to C# tutorial: Collections
```

Comment:
nit: should H1 header match the "title" in the metadata? 
I thought it's a guideline.

Reply:
@pkulikov  No, the title should not match. Having different titles helps get this article pulled into more results.

Reply:
@Thraka cool! Good to know; thanks for the clarification.